src/modules/tests/examiner/examiner.selector.ts

```diff
@@ -0,0 +1,3 @@
+import { Examiner } from '@dvsa/mes-test-schema/categories/B';
+
+export const getStaffNumber = (examiner: Examiner) => examiner.staffNumber || '';
```

Comment:
Assuming the default is just to keep the compiler happy?

Reply:
believe there was an issue when running in web mode.